src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs

```diff
@@ -273,15 +272,15 @@ public void should_not_scan_subfolders_that_start_with_period()
         [Test]
         public void should_not_scan_subfolder_of_season_folder_that_starts_with_a_period()
         {
-            GivenSeriesFolder();
+            GivenArtistFolder();
 
             GivenFiles(new List<string>
                        {
-                           Path.Combine(_artist.Path, "Season 1", ".@__thumb", "file1.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", ".@__THUMB", "file2.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", ".hidden", "file2.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", ".AppleDouble", "s01e01.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", "s01e01.mkv").AsOsAgnostic()
+                           Path.Combine(_artist.Path, "Season 1", ".@__thumb", "file1.flac").AsOsAgnostic(),
+                           Path.Combine(_artist.Path, "Season 1", ".@__THUMB", "file2.flac").AsOsAgnostic(),
+                           Path.Combine(_artist.Path, "Season 1", ".hidden", "file2.flac").AsOsAgnostic(),
+                           Path.Combine(_artist.Path, "Season 1", ".AppleDouble", "s01e01.flac").AsOsAgnostic(),
```

Comment:
Just noticed this. Mac also likes to create files called `.DS_STORE`. Do we have anything to detect those files and handle accordingly.

Reply:
added as a test case. Lidarr should ignore as its not a recognized audio or extras extension.